src/snapred/backend/data/LocalDataService.py

```diff
@@ -303,230 +253,85 @@ def _constructReductionDataRoot(self, runNumber: str, useLiteMode: bool) -> Path
-            relevantEntries = list(filter(lambda x: self._isApplicableEntry(x, runId), calibrationIndex))
-            if len(relevantEntries) < 1:
-                return None
-            latestCalibration = relevantEntries[-1]
-            version = latestCalibration.version
-        return version
-
-    def _getVersionFromNormalizationIndex(self, runId: str, useLiteMode: bool) -> int:
-        """
-        Loads normalization index and inspects all entries to attain
-        latest normalization version that applies to the run id
-        """
-        # lookup normalization index
-        normalizationIndex = self.readNormalizationIndex(runId, useLiteMode)
-        # From the index find the latest normalization
-        latestNormalization = None
-        version = None
-        if normalizationIndex:
-            # sort by timestamp
-            normalizationIndex.sort(key=lambda x: x.timestamp)
-            # filter for latest applicable
-            relevantEntries = list(filter(lambda x: self._isApplicableEntry(x, runId), normalizationIndex))
-            if len(relevantEntries) < 1:
-                return None
-            latestNormalization = relevantEntries[-1]
-            version = latestNormalization.version
-        return version
-
-    def getVersionFromNormalizationIndex(self, runId: str, useLiteMode: bool) -> int:
-        return self._getVersionFromNormalizationIndex(runId, useLiteMode)
-
-    def writeCalibrationIndexEntry(self, entry: CalibrationIndexEntry):
-        stateId, _ = self._generateStateId(entry.runNumber)
-        calibrationPath: Path = self._constructCalibrationStatePath(stateId, entry.useLiteMode)
-        indexPath: Path = calibrationPath / "CalibrationIndex.json"
-        # append to index and write to file
-        calibrationIndex = self.readCalibrationIndex(entry.runNumber, entry.useLiteMode)
-        calibrationIndex.append(entry)
-        write_model_list_pretty(calibrationIndex, indexPath)
-
-    def writeNormalizationIndexEntry(self, entry: NormalizationIndexEntry):
-        stateId, _ = self._generateStateId(entry.runNumber)
-        normalizationPath: Path = self._constructNormalizationStatePath(stateId, entry.useLiteMode)
-        indexPath: Path = normalizationPath / "NormalizationIndex.json"
-        # append to index and write to file
-        normalizationIndex = self.readNormalizationIndex(entry.runNumber, entry.useLiteMode)
-        normalizationIndex.append(entry)
-        write_model_list_pretty(normalizationIndex, indexPath)
-
-    @validate_call
-    def getCalibrationRecordFilePath(self, runId: str, useLiteMode: bool, version: Version) -> Path:
-        return self._constructCalibrationDataPath(runId, useLiteMode, version) / "CalibrationRecord.json"
-
-    @validate_call
-    def getNormalizationRecordFilePath(self, runId: str, useLiteMode: bool, version: Version) -> Path:
-        return self._constructNormalizationDataPath(runId, useLiteMode, version) / "NormalizationRecord.json"
```

Comment:
These were not called anywhere, and can be replaced with `indexor(etc).recordPath(version)`.